kyo-data/shared/src/main/scala/kyo/Text.scala

```diff
@@ -7,6 +7,8 @@ opaque type Text >: String = String | Op
 
 object Text:
 
+    given Tag[Text] = Tag.fromRaw("kyo.Text")
```

Comment:
A workaround since it's currently not possible to have a `Tag` for a type union. This raw tag will only support exact match and not `<:<`, which is ok since that's what effect handling uses.